frontend/locales/en/strings.json

```diff
@@ -187,6 +187,16 @@
         "scaledownIdleTime": {
           "label": "Scaledown Idle Time (minutes)",
           "placeholder": "10"
+        },
+        "queueUpdateStrategy": {
+          "label": "Queue Update Strategy",
+          "placeholder": "Select a queue update strategy",
+          "selectedAriaLabel": "Selected",
+          "options": {
+            "computeFleetStop": "COMPUTE_FLEET_STOP",
```

Comment:
thought: do we want to keep labels in the same YAML format?

Otherwise I was thinking of something like that:

```
Compute Fleet Stop
Drain
Terminate
```

Reply:
It's a fair point, to be consistent with other `Select`s it's probably better if I use a human readable label

Will fix